worker/src/RTC/SvcConsumer.cpp

```diff
@@ -539,6 +539,14 @@ namespace RTC
 		if (!IsActive())
 			return;
 
+		// Packets with only padding are not forwarded.
+		if (packet->GetPayloadLength() == 0)
+		{
+			this->rtpSeqManager.Drop(packet->GetSequenceNumber());
+
+			return;
+		}
```

Comment:
This is missing in `PipeConsumer.cpp`.

Reply:
@ggarber this is missing. `PipeConsumer.cpp` does exist and requires the same changes.

Reply:
reminder of this missing piece

Reply:
Oh no, I see it done.